src/main/python/RLBotFramework/utils/rendering/rendering_manager.py

```diff
@@ -0,0 +1,199 @@
+import collections
+import ctypes
+
+from RLBotFramework.utils.structures.game_status import RLBotCoreStatus
+from RLBotFramework.utils.logging_utils import get_logger
+from RLBotFramework.utils.structures.game_data_struct import Vector3
+
+
+class Color(ctypes.Structure):
+    _fields_ = [("b", ctypes.c_char),
+                ("g", ctypes.c_char),
+                ("r", ctypes.c_char),
+                ("a", ctypes.c_char)]
+
+
+class RenderingManager:
+    game = None
+    render_state = False
+    ignored_funcs = []
+
+    def __init__(self):
+        self.ignored_funcs = ['setup_function_types', 'get_render_functions',
+                              'create_dynamic_function', 'send_group']
+
+    def setup_function_types(self, dll_instance):
+        self.game = dll_instance
+
+        func = self.game.DrawLine2D

```

Comment:
Why is this needed? Don't we use RenderGroup for everything?

Reply:
Removed